host-configs/TOTAL/pangea3-gcc8.3-smpi-10.3.0.1.cmake

```diff
@@ -0,0 +1,121 @@
+# hostconfig for pangea3
+# module load cmake/3.18.2 gcc/8.3.0 cuda/11.3.0 smpi/10.3.0.1 openblas/0.3.8 lapack/3.9.0
+set(CONFIG_NAME "pangea3-gcc8.3-smpi-10.3.0.1" CACHE PATH "") 
+
+# Set up the tpls
+if (DEFINED ENV{GEOSX_TPL_DIR})
+  set(GEOSX_TPL_DIR ENV{GEOSX_TPL_DIR})
+else ()
+  message(FATAL_ERROR "You must set GEOSX_TPL_DIR")
+endif ()
+
+if ((DEFINED ENV{CC}) AND (DEFINED ENV{CXX}) AND (DEFINED ENV{FC}))
+  # C options
+  set(CMAKE_C_COMPILER $ENV{CC} CACHE PATH "")
+  set(CMAKE_C_FLAGS_RELEASE "-O3 -DNDEBUG -mcpu=power9 -mtune=power9" CACHE STRING "")
+  set(CMAKE_C_FLAGS_RELWITHDEBINFO "-g ${CMAKE_C_FLAGS_RELEASE}" CACHE STRING "")
+  set(CMAKE_C_FLAGS_DEBUG "-O0 -g" CACHE STRING "")
+
+  # C++ options
+  set(CMAKE_CXX_COMPILER $ENV{CXX} CACHE PATH "")
+  set(CMAKE_CXX_FLAGS_RELEASE "-O3 -DNDEBUG -mcpu=power9 -mtune=power9" CACHE STRING "")
+  set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-g ${CMAKE_CXX_FLAGS_RELEASE}" CACHE STRING "")
+  set(CMAKE_CXX_FLAGS_DEBUG "-O0 -g" CACHE STRING "")
+  set(CMAKE_CXX_STANDARD 14 CACHE STRING "")
+
+  # Fortran options
+  set(CMAKE_Fortran_COMPILER $ENV{FC} CACHE PATH "")
+  set(CMAKE_Fortran_FLAGS_RELEASE "-O3 -DNDEBUG -mcpu=power9 -mtune=power9" CACHE STRING "")
+  #set(FORTRAN_MANGLE_NO_UNDERSCORE ON CACHE BOOL "")
+else()
+  message(FATAL_ERROR "You must load module gcc/8.3")
```

Comment:
Note that this will not ensure that the user has the proper `gcc` version loaded.

Reply:
Indeed i just wanted to check that the ENV i use where defined. One Can use another one but my message indicate the version i tested. Different MPI Can generate hdf5 errors for example. But other might work.

Reply:
Maybe i could change my message to "please define CC,CCX,FC variables. We recommand loading module...".

Reply:
So maybe comparing with the exact path of the `CC` you want could be a good way to enforce the version?

Reply:
> Maybe i could change my message to "please define CC,CCX,FC variables. We recommand loading module...".

It's your users 😉